VimeoNetworkingExample-iOS/VimeoNetworkingExample-iOS/MasterViewController.swift

```diff
@@ -8,95 +8,172 @@
 
 import UIKit
 
-class MasterViewController: UITableViewController {
-
+class MasterViewController: UITableViewController
+{
     var detailViewController: DetailViewController? = nil
-    var objects = [AnyObject]()
-
-
-    override func viewDidLoad() {
+    var videos: [VIMVideo] = []
+    {
+        didSet
+        {
+            self.tableView.reloadData()
+        }
+    }
+    
+    private var accountObservationToken: ObservationToken?
+    private var authenticationButton: UIBarButtonItem?
+    
+    // MARK: - View Controller
+
+    override func viewDidLoad()
+    {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
-        self.navigationItem.leftBarButtonItem = self.editButtonItem()
-
-        let addButton = UIBarButtonItem(barButtonSystemItem: .Add, target: self, action: #selector(insertNewObject))
-        self.navigationItem.rightBarButtonItem = addButton
-        if let split = self.splitViewController {
+        
+        if let split = self.splitViewController
+        {
```

Comment:
seems here we if-let unwrap but below force in `self.splitViewController!.collapsed`